src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,23 @@
+.stars {
+  width: 96px;
```

Comment:
According to the [Code Style Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), the width property should not be fixed for blocks. Consider using max-width if necessary, or ensure that the width is flexible and responsive.